docs/ui-ux/README.md

```diff
@@ -0,0 +1,105 @@
+# User Interface and User Experience Engineering
+
+Also known as _UI/UX_, _Front End Development_, or _Web Development_, user interface and user experience engineering is a broad topic and encompasses many different aspects of modern application development. When a user interface is required, CSE primarily develops a **web application**. Web apps can be built in a variety of ways with many different tools.
+
+## Goal
+
+The goal of the **User Interface Engineering Fundamentals** group is to provide guidance on developing web applications for CSE engagements. Everyone should begin by reading the [General Guidance](#general-guidance) for a quick introduction to the four main aspects of every web application project. From there, readers are encouraged to dive deeper into each topic or begin reviewing technical guidance that pertains to their engagement. All UI/UX projects should begin with a detailed design document. Review the [Design Document](#design-document) section for more details and a template to get your crew started.
```

Comment:
```suggestion
The goal of the **User Interface** section is to provide guidance on developing web applications for CSE engagements. Everyone should begin by reading the [General Guidance](#general-guidance) for a quick introduction to the four main aspects of every web application project. From there, readers are encouraged to dive deeper into each topic, or begin reviewing technical guidance that pertains to their engagement. All UI/UX projects should begin with a detailed design document. Review the [Design Document](#design-document) section for more details, and a template to get started.
```

We should see the playbook as standalone from the EF groups - and ideally shouldn't reference internal groups or processes. The playbook should ideally be readable and applicable also to teams outside of CSE.